front-end/src/Views/ProjectPage.js

```diff
@@ -32,7 +34,7 @@ const Container = styled(Box)({
   }
 });
 
-export default class HomePage extends Component {
+export default class EditProjectPage extends Component {

```

Comment:
Since we are in ProjectPage I think this should be ProjectPage not EditProjectPage